tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -22,16 +22,27 @@ pr:
       - tools/apiview/parsers/cpp-api-parser
 
 variables:
-  ArtifactName: 'apiview'
-  VcpkgRelease: '2024.08.23'
+  - template: /eng/pipelines/templates/variables/image.yml
+  - template: /eng/pipelines/templates/variables/globals.yml
+  - name: ArtifactName
+    value: 'apiview'
+  - name: VcpkgRelease
+    value: '2024.08.23'
+
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
```

Comment:
Also include the ref https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/templates/stages/1es-redirect.yml#L10 so that these versions remain in sync.